### DISCUSSION TITLE
ترجمة مصطلح Implementation

### DISCUSSION THREAD

**Discussion by boussadjra** [vuejs-translations/docs-ar#81] [مصطلحات]
من خلال المترجمات الآلية وجدت أنها تعني تنفيذ، لكن بدت لي لا تفي بالمعنى و قد اقترحت مؤقتا ترجمتها للشيفرة التنفيذية.

**Comment by Abdelaziz18003** [comment]
أرى أنها تعني معانٍ كثيرة متقاربة وترجمتها قد تختلف بحسب السياق الذي ترد فيه، الترجمة المؤقتة التي اقترحتها قريبة، لنستعملها مؤقتا ريثما نجد ترجمات أدق 👍

**Comment by boussadjra** [comment]
ما قصدته هو في سياق الحديث عن شيفرة الدالة
